include/firefly/memory-manager/mm.hpp

```diff
@@ -9,6 +9,12 @@ enum AddressLayout : uint64_t {
     Low = 0x0000000000000000UL
 };
 
+enum PageSize : uint64_t {
+	SIZE_4KB = 0x00001000,
```

Comment:
I know we don't have a code style guideline, that's totally on me sorry.
It would be `Size4K`, `Size1GB`, etc.